### PR TITLE
User story 9 walkable player

### PR DESCRIPTION
**Pull request by HugoUnosson**

### CODE REVIEW

**Comment by Caramajau** [comment]
Game crashed after fishing once.
game_crash

**Comment by Caramajau** [comment]
> Game crashed after fishing once. game_crash

It seems like it is inconsitent and only happens when some kind of combination of inputs is done.

File: DAT257AgileProject/Assets/Scripts/Fishing/FishingSpot.cs [hunk 1]
```diff
@@ -11,13 +11,11 @@
...
 
-    [SerializeField] 
-    private Sprite fishingSprite1, fishingSprite2;
     [SerializeField] 
     private GameObject exclamationMarkPrefab;
 
-    private static bool isFishing = true;
-    private bool isPlayingMinigame = false;
+    public static bool isFishing = true;
+    public bool isPlayingMinigame { get; private set; }
```
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scripts/Fishing/FishingSpot.cs:18]: Why is "isFishing" public? and if it has to be at least use a setter or property.

Comment (HugoUnosson) [reply DAT257AgileProject/Assets/Scripts/Fishing/FishingSpot.cs:18]: Just forgot to change it back to private

**Review by Caramajau — CHANGES_REQUESTED** [review]
See comment about public field, saw now that the game crash exists in sprint_3 as well.

**Comment by HugoUnosson** [comment]
Updated the sprites for minigama and exclamation mark

**Comment by HugoUnosson** [comment]
> See comment about public field, saw now that the game crash exists in sprint_3 as well.

And btw I did not try to fix the bug since it probably has to do with something else than animations??

**Review by Caramajau — APPROVED** [review]
For me it's fine now.

**Comment by Caramajau** [comment]
> I think it is well implemented except for some bugs. One thing is that you start fishing without evan pressing f (maybe its a feature i dont know). Also when I catch something and dont close the info window and start fishing again I get stuck and cant get out of fishing or the info screen.

I believe it is intentional that you fish when you touch the water tile, don't know anything about the other bug though.

**Comment by HugoUnosson** [comment]
> I think it is well implemented except for some bugs. One thing is that you start fishing without evan pressing f (maybe its a feature i dont know). Also when I catch something and dont close the info window and start fishing again I get stuck and cant get out of fishing or the info screen.

Yes it is a feature. Dont know why you get stuck, it seems to happen in other branches too

**Review by HermanForsberg — APPROVED** [review]

**Pull request merged**